tuxemon/db.py

```diff
@@ -254,6 +261,13 @@ class MonsterEvolutionItemModel(BaseModel):
     gender: Optional[GenderType] = Field(None, description="Gender parameter")
     mixed: Optional[str] = Field(None, description="Mixed parameter.")
 
+    @validator("at_level")
+    def is_multiple(cls, v):
+        if v > 0:
+            if has.multiple(v, 3):
+                return v
+            raise ValueError(f"no resource exists in db: {v}")
```

Comment:
this error message is misleading

Reply:
I'm going to remove it, I used it for maintaining order